wpimath/src/main/native/include/frc/kinematics/ChassisSpeeds.h

```diff
@@ -38,6 +39,27 @@ struct WPILIB_DLLEXPORT ChassisSpeeds {
    */
   units::radians_per_second_t omega = 0_rad_per_s;
 
+  /**
+   * Returns a chassis speed that, after a given duration, will produce this
+   * chassis speed scaled by the duration.
```

Comment:
This comment doesn't seem accurate. What it describes is just dividing the existing chassis speed by dt, which isn't what's happening.

I'm trying to come up with a better function name.